newswires/client/src/WireDetail.tsx

```diff
@@ -276,65 +282,47 @@ export const WireDetail = ({
 					</h3>
 					<EuiSpacer size="s" />
 					{ednote && <EuiCallOut size="s" title={ednote} color="success" />}
-					<EuiSpacer size="s" />
+					<EuiSpacer size="m" />
```

Comment:
bit tangential to this PR, but should one of these spacers be inside the `{ednote` conditional block? otherwise I imagine we end up with a double spacer

Reply:
good call, cheers! I've fixed that now 👍